bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -8116,14 +8130,14 @@ components:
           description: |
             Number of confirmed equipment units
           example: 3
-        emptyContainerPositionings:
+        containerPositionings:
           type: array
           description: |
-            List of `ECP` (Empty Container Positioning) locations.
+            List of `CPO` (Container Positioning) locations.
```

Comment:
Shouldn't be the description as same as IFS one?

Reply:
Text is synced with the Requested Equipment + added that it is estimated date time